### PR TITLE
[DX-1358] Dashboard release notes 5.4.0

### PR DESCRIPTION
**Pull request by Eopayemi**
### For internal users - Please add a Jira DX PR ticket to the subject!

[DX-1358](https://tyktech.atlassian.net/browse/DX-1358?atlOrigin=eyJpIjoiNDQ4NGZjODk2NzVmNDYxNjlkMzBlNDk0MDYzODk2MzMiLCJwIjoiamlyYS1zbGFjay1pbnQifQ)

---

### Preview Link

[preview](https://deploy-preview-4663--tyk-docs.netlify.app/docs/nightly/product-stack/tyk-dashboard/release-notes/version-5.4)

### Description

Dashboard release notes 5.4.0, build checks will be fixed when Tyk Charts 1.5.0 release notes merged.

#### Screenshots (if appropriate)
<br>

## Checklist

- [x] I have added a **preview link** to the PR description.
- [x] I have [reviewed the guidelines](https://github.com/TykTechnologies/tyk-docs/blob/master/CONTRIBUTING.md) for contributing to this repository.
- [x] I have [read the technical guidelines](https://github.com/TykTechnologies/tyk-docs/blob/master/CONTRIBUTING-TECHNICAL-GUIDE.md) for contributing to this repository.
- [x] Make sure you have started *your change* off *our latest `master`*.
- [x] I **labelled** the PR

___

### **PR Type**
Documentation

___

### **Description**
- Added comprehensive release notes for Tyk Dashboard version 5.4.0.
- Included detailed sections for breaking changes, dependencies, deprecations, upgrade instructions, release highlights, downloads, changelog, and security fixes.
- Provided compatibility matrix for Tyk components and third-party dependencies.
- Added further information section with guidance on upgrading Tyk, links to API documentation, and FAQ.

___

### **Changes walkthrough** 📝
<table><thead><tr><th></th><th align="left">Relevant files</th></tr></thead><tbody><tr><td><strong>Documentation
</strong></td><td><table>
<tr>
  <td>
    <details>
      <summary><strong>version-5.4.md</strong><dd><code>Added Tyk Dashboard 5.4.0 release notes documentation.</code>&nbsp; &nbsp; &nbsp; </dd></summary>
<hr>

tyk-docs/content/product-stack/tyk-dashboard/release-notes/version-5.4.md
<li>Added release notes for Tyk Dashboard version 5.4.0.<br> <li> Included sections for breaking changes, dependencies, deprecations, <br>upgrade instructions, release highlights, downloads, changelog, and <br>security fixes.<br> <li> Provided compatibility matrix and third-party dependencies.<br> <li> Added further information section with links to upgrading Tyk, API <br>documentation, and FAQ.<br>

</details>
    

  </td>
  <td><a href="https://github.com/TykTechnologies/tyk-docs/pull/4663/files#diff-8b10e0595c9b19f13fab9cafc0ba04ff18b88a3dc111f20e38924741855ccdbf">+193/-0</a>&nbsp; </td>
</tr>                    
</table></td></tr></tr></tbody></table>

___

> 💡 **PR-Agent usage**:
>Comment `/help` on the PR to get a list of all available PR-Agent tools and their descriptions

[DX-1358]: https://tyktech.atlassian.net/browse/DX-1358?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: tyk-docs/content/product-stack/tyk-dashboard/release-notes/version-5.4.md [hunk 1]
```diff
@@ -0,0 +1,207 @@
...
+### Dependencies {#dependencies-5.4.0}
+<!--Required. Use this section to announce the following types of dependencies compatible with the release:
+Version compatibility with other components in the Tyk stack. This takes the form of a compatibility matrix and is only required for Gateway and Portal.
+3rd party dependencies and tools -->
+#### Compatibility Matrix For Tyk Components
+<!-- Required. Version compatibility with other components in the Tyk stack. This takes the form of a compatibility matrix and is only required for Gateway and Portal.
+An illustrative example is shown below. -->
+| Dashboard Version | Recommended Releases | Backwards Compatibility |
+|----    |---- |---- |
+| 5.3.1 | MDCB v2.5.1     | MDCB v2.5.1 |
```
Comment (carolwkc) [inline tyk-docs/content/product-stack/tyk-dashboard/release-notes/version-5.4.md]: For Dashboard 5.4, please update to
- MDCB v2.6.0
- Operator 0.18
- Sync 1.5
- Helm chart 1.5
- Pump 1.10

File: tyk-docs/content/product-stack/tyk-dashboard/release-notes/version-5.4.md [hunk 2]
```diff
@@ -0,0 +1,207 @@
...
+Additionally, a disclaimer statement was added below the table, for customers to check that the third-party dependency they decide to install remains in support.
+
+An example is given below for illustrative purposes only. Tested Versions and Compatible Versions information will require discussion with relevant squads and QA. -->
+
+| Third Party Dependency                                     | Tested Versions        | Compatible Versions    | Comments | 
+| ---------------------------------------------------------- | ---------------------- | ---------------------- | -------- | 
+| [GoLang](https://go.dev/dl/)                               | 1.21       | 1.21       | [Go plugins]({{< ref "plugins/supported-languages/golang" >}}) must be built using Go 1.21 | 
+| [Redis](https://redis.io/download/)  | 6.2.x, 7.x  | 6.2.x, 7.x  | Used by Tyk Dashboard | 
+| [MongoDB](https://www.mongodb.com/try/download/community)  | 5.0.x, 6.0.x, 7.0.x  | 5.0.x, 6.0.x, 7.0.x  | Used by Tyk Dashboard | 
+| [PostgreSQL](https://www.postgresql.org/download/)         | 11.x - 15.x LTS        | 11.x - 15.x            | Used by Tyk Dashboard |
```
Comment (carolwkc) [inline tyk-docs/content/product-stack/tyk-dashboard/release-notes/version-5.4.md]: PostgreSQL 12.x - 16.x

File: tyk-docs/content/product-stack/tyk-dashboard/release-notes/version-5.4.md [hunk 3]
```diff
@@ -0,0 +1,207 @@
...
+-->
+**Add release highlights here**
+
+### Downloads
+- [Docker Image to pull](https://hub.docker.com/r/tykio/tyk-dashboard/tags?page=&page_size=&ordering=&name=v5.4.0)
+- ```bash
+  docker pull tykio/tyk-dashboard:v5.4.0
+  ```
+- Helm charts
+  - [tyk-charts v1.3]({{< ref "product-stack/tyk-charts/release-notes/version-1.3.md" >}})
```
Comment (carolwkc) [inline tyk-docs/content/product-stack/tyk-dashboard/release-notes/version-5.4.md]:
```suggestion
  - [tyk-charts v1.5]({{< ref "product-stack/tyk-charts/release-notes/version-1.5.md" >}})
```

**Review by dcs3spp — CHANGES_REQUESTED** [review]
PR LGTM, with following minor suggestions:

@agata-wit @andyo-tyk @caroltyk @slega Could you check and update compatibility matrix, 3rd party dependencies and upgrade section in relation to release 5.4.0
@Eopayemi Could you add change log and release highlights (DONE)

File: tyk-docs/content/product-stack/tyk-dashboard/release-notes/version-5.4.md [hunk 4]
```diff
@@ -0,0 +1,251 @@
...
+<!-- Required. oss or licensed. Choose one of the following:
+    **Licensed Protected Product**
+    Or
+    ****Open Source** ([Mozilla Public License](https://github.com/TykTechnologies/tyk/blob/master/LICENSE.md))**
+-->
+**Open Source** ([Mozilla Public License](https://github.com/TykTechnologies/tyk/blob/master/LICENSE.md))
+**This page contains all release notes for version 5.4.X displayed in a reverse chronological order**
+## Support Lifetime
+<!-- Required. replace X.Y with this release and set the correct quarter of the year -->
+Our minor releases are supported until our next minor comes out.
```
Comment (andyo-tyk) [inline tyk-docs/content/product-stack/tyk-dashboard/release-notes/version-5.4.md:17]: Not sure why, but this is appearing as a title in the preview

**Review by andyo-tyk — APPROVED** [review]

File: tyk-docs/content/product-stack/tyk-dashboard/release-notes/version-5.4.md (hunk 4 above)
Comment (dcs3spp) [reply tyk-docs/content/product-stack/tyk-dashboard/release-notes/version-5.4.md:17]: Thanks @andyo-tyk good spot, I've updated and once preview generated should be fixed

**Review by carolwkc — APPROVED** [review]
table updated

**Review by agata-wit — APPROVED** [review]

**Review by dcs3spp — APPROVED** [review]

**Pull request merged**